disdat/dsdt.py

```diff
@@ -104,7 +104,7 @@ def main():
     apply_p.add_argument('-f', '--force', action='store_true', help="If there are dependencies, force re-computation.")
     apply_p.add_argument('--incremental-push', action='store_true', help="Commit and push each task's bundle as it is produced to the remote.")
     apply_p.add_argument('--incremental-pull', action='store_true', help="Localize bundles as they are needed by downstream tasks from the remote.")
-    apply_p.add_argument('pipe_cls', type=str, help="User-defined transform, e.g., 'module.PipeClass'")
+    apply_p.add_argument('pipe_cls', type=load_class, help="User-defined transform, e.g., 'module.PipeClass'")
```

Comment:
Is it pretty to have a type be a method?  It's opaque.  Can we leave at as str and do this in apply.main() and maybe rename main?  Note that, b/c we expressly called task_register code previously, we no longer get a list of possibly task candidates as we once did when someone enters in a bogus value.    
`dsdt apply swordfish.monger` used to give:
```
luigi.task_register.TaskClassNotFoundException: No task swordfish. Candidates are: BundleWrapperTask,Config,DriverTask,ExternalTask,PipeTask,RangeBase,RangeByMinutes,RangeByMinutesBase,RangeDaily,RangeDailyBase,RangeHourly,RangeHourlyBase,RangeMonthly,Task,TestNotificationsTask,WrapperTask,batch_email,core,email,execution_summary,scheduler,sendgrid,smtp,worker
```
but now gives
`ModuleNotFoundError: No module named 'swordfishasd'`